tests/foreman/cli/test_partitiontable.py

```diff
@@ -76,6 +79,21 @@ def test_positive_create_with_content(self):
         ptable_content = PartitionTable().dump({'id': ptable['id']})
         self.assertTrue(content in ptable_content[0])
 
+    @tier1
+    def test_positive_create_with_content_length(self):
+        """Create a Partition Table with content length more than 4096 chars
+
+        :id: 59e6f9ef-85c2-4229-8831-00edb41b19f4
+
+        :expectedresults: Partition Table is created and has correct content
+
+        :BZ: 1270181
+        """
+        content = gen_string('alpha', 5000)
+        ptable = make_partition_table({'content': content})
+        ptable_content = PartitionTable().dump({'id': ptable['id']})
```

Comment:
so you read all partition tables and then take one entity? Could we do using any application methods like `search`?
Do we have BZ about `content` is not returned for usual read operation?